src/types/tables.ts

```diff
@@ -14,8 +14,8 @@ export interface DataTableMetaData {
 }
 
 export interface FormattedData {
-  data: DataTableDataType[];
-  headers: DataTableHeader[];
+  data: DataTableDataType[] | [];
+  headers: DataTableHeader[] | [];
```

Comment:
Is this needed? What error do you get if you don't add this?

Reply:
It's if there's no data passed, this is required to return an empty dataset rather than undefined. I'll put a condition in once we get designs for if there's no data we can display a message or empty table, whatever design come back with...